src/docs/Components/Cards/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Cards'
+order: 1
+description: 'A card is a graphic element that structures a group of related informations that must be understood on its own. It visually looks like a playing card.'
+status:
+  sketch: 'wip'
+  scss: 'wip'
+---
+
+> A card is a graphic element that structures a group of related informations that must be understood on its own. It visually looks like a playing card.
+
+## Usages
+
+A card is mostly a visualisation of the page that is to come behind the card itself.
+
+-- Add a preview of the two card versions next to each other. --
+
+### Outlined version
+
+Outlined cards are used to avoid flotting effects, for exemple when you create parallax or depth effects with differents size of cards.
```

Comment:
Outlined cards are used to avoid floating impressions, for example, when you create parallax or depth effects with different card sizes.